gtsfm/utils/metrics.py

```diff
@@ -513,6 +513,30 @@ def get_measurement_angle_errors(
     return errors
 
 
+def pose_auc_from_poses(
+    computed_wTis: List[Pose3], ref_wTis: List[Pose3], thresholds_deg: Sequence[float]
+) -> Sequence[float]:
+    """Computes area under the Recall (y) vs. Pose Error (x) curve, the pose AUC.
+
+    If recall is defined as TP / # actual positives, then every camera is a TP if one can register it.
+
+    Args:
+        computed_wTis: list of poses for comparison, of length N.
+        ref_wTis: references list of poses, of length N.
```

Comment:
worth noting?

    Note: inputs must be aligned beforehand to the ground truth.